From 16d318c502d17a4c516c42fa9f2af0cba2909d7a Mon Sep 17 00:00:00 2001
From: CanYang He <hcy@rock-chips.com>
Date: Thu, 2 Aug 2018 17:49:08 +0800
Subject: [PATCH] arm64: dts: rockchip: prevent some lp4 scale frequency for
 rk3399 evb3 lp4

lpddr4 scale frequency when display on, cause the display shake and
vop POST_BUF_EMPTY

Change-Id: I2bfef85ae34c3c85609dcb3506272ff0e27cb0c7
Signed-off-by: CanYang He <hcy@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-lp4.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-lp4.dts b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-lp4.dts
index 8dd0a0bf36ee..3afffea7b071 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-lp4.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-lp4.dts
@@ -117,11 +117,11 @@
 		SYS_STATUS_NORMAL       800000
 		SYS_STATUS_REBOOT       400000
 		SYS_STATUS_SUSPEND      400000
-		SYS_STATUS_VIDEO_1080P  400000
+		SYS_STATUS_VIDEO_1080P  800000
 		SYS_STATUS_VIDEO_4K     800000
 		SYS_STATUS_VIDEO_4K_10B 800000
 		SYS_STATUS_PERFORMANCE  800000
-		SYS_STATUS_BOOST        400000
+		SYS_STATUS_BOOST        800000
 		SYS_STATUS_DUALVIEW     800000
 		SYS_STATUS_ISP          800000
 	>;
-- 
2.35.3

